### PR TITLE
Added organiser user story

### PR DESCRIPTION
**Pull request by shangeethsivan**
Added attendee check-in feature to the organiser user story. As a event volunteer I badly needed a check in app. I have volunteered many events and a check in app is a big missing part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,9 @@ So far we have only two user roles: participants and group organizers
 
 - I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
 
-I can see how many times a participant has come to the event as well as their attendance rate
+- I can see how many times a participant has come to the event as well as their attendance rate
```
Comment (francocorreasosa) [inline README.md]: Not really sure this is something we need for the MVP. Let's raise it in Discord 😄

Comment (shangeethsivan) [reply README.md]: @francocorreasosa Sorry this was merged  I just added for matting to it.

**Comment by vkWeb** [comment]
@francocorreasosa This is definitely something we should keep on the MVP. I have experienced the same situation in many events. So let's keep this on MVP.

@QuincyLarson what do you think?

File: README.md (hunk 1 above)
Comment (MirzaChilman) [reply README.md]: That's my changes silly me forgot about the bulleting

Anyway it's a good features that we need for MVP, the current schema also has already adjusted with this story

**Comment by nik-john** [comment]
@all-contributors please add @shangeethsivan to doc

**Pull request merged**